frontend/src/constants/reducerTypes.tsx

```diff
@@ -8,3 +8,6 @@ export const GET_PARCEL_DETAIL = 'GET_PARCEL_DETAIL';
 // Lookup Codes
 export const LOOKUP_CODE = 'lookupCode';
 export const GET_LOOKUP_CODES = 'GET_LOOKUP_CODES';
+
+// Auth
+export const POST_ACTIVATE_USER = "activateUser";
```

Comment:
I'm curious to learn how the automatic linting didn't kick in here as one of the rules we have is to use single quotes instead of double quotes. That's something the linter is supposed to auto-fix for us before we are able to commit changes. Did you run `npm install` on your local copy? Note that my PR added new dependencies...

Reply:
I ran npm ci. perhaps merged changes are exempt?